pages/api/v1/user-created-bounties/submissions/editOrDeleteSubmission.ts

```
@@ -0,0 +1,107 @@
+
+		const { title, content, tags, link, reqAmount, proposerAddress, parentBountyIndex, submissionId, action } = req.body;
+		const encodedProposerAddress = getEncodedAddress(proposerAddress, network);
+
+		if (![EUserCreatedBountyActions.EDIT, EUserCreatedBountyActions.DELETE].includes(action)) {
+			return res.status(400).json({ message: 'Invalid Action Param' });
+		}
+		if (!proposerAddress?.length || !encodedProposerAddress) {
+			return res.status(400).json({ message: 'Invalid Proposer Address' });
+		}
+
+		if (isNaN(parentBountyIndex)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)